src/Bridge/Doctrine/Orm/Filter/BooleanFilter.php

```diff
@@ -50,18 +54,29 @@ public function apply(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $q
 
         $properties = $this->extractProperties($request);
 
-        foreach ($properties as $property => $boolean) {
-            if (!$this->isPropertyEnabled($property) || !$this->isPropertyMapped($property, $resourceClass)) {
+        foreach ($properties as $property => $value) {
+            if (
+                !$this->isPropertyEnabled($property) ||
+                !$this->isPropertyMapped($property, $resourceClass) ||
+                !$this->isBooleanField($property, $resourceClass)
+            ) {
                 continue;
             }
 
-            if (empty($boolean) && isset($this->properties[$property])) {
-                $boolean = $this->properties[$property];
-            }
-
-            $filterBoolean = filter_var($boolean, FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE);
```

Comment:
I missed this change before. What is the rationale behind not using this filter?
It complexifies the code and is less intuitive for PHP developers.

Even if I never use `yes` and `no` myself, I think it doesn't hurt.